Orange/widgets/unsupervised/tests/test_owkmeans.py

```diff
@@ -0,0 +1,43 @@
+from PyQt4 import QtGui
+
+from Orange.widgets.tests.base import WidgetTest
+from Orange.widgets.unsupervised.owkmeans import OWKMeans
+
+from Orange.data import Table
+class TestOWKMeans(WidgetTest):
+
+    def setUp(self):
+        self.widget = self.create_widget(OWKMeans)
+        self.combo_box = self.widget.findChildren(QtGui.QComboBox)
```

Comment:
A better way would be to modify the widget to store qt components to instance attributes and then use those. That way, the test would still work if new controls of the same type are added to the widget.